pymc3/distributions/multivariate.py

```diff
@@ -1091,3 +1091,212 @@ def logp(self, x):
+    rowcov : mxm array
+        Among-row covariance matrix. Defines variance within
+        columns. Exactly one of rowcov or rowchol is needed.
+    rowchol : mxm array
+        Cholesky decomposition of among-row covariance matrix. Exactly one of
+        rowcov or rowchol is needed.
+    colcov : nxn array
+        Among-column covariance matrix. If rowcov is the identity matrix,
+        this functions as `cov` in MvNormal.
+        Exactly one of colcov or colchol is needed.
+    colchol : nxn array
+        Cholesky decomposition of among-column covariance matrix. Exactly one
+        of colcov or colchol is needed.
+
+    Examples
+    --------
+    Define a matrixvariate normal variable for given row and column covariance
+    matrices::
+
+        colcov = np.array([[1., 0.5], [0.5, 2]])
+        rowcov = np.array([[1, 0, 0], [0, 4, 0], [0, 0, 16]])
+        m = rowcov.shape[0]
+        n = colcov.shape[0]
+        mu = np.zeros((q, p))
+        vals = pm.MatrixNormal('vals', mu=mu, colcov=colcov,
+                               rowcov=rowcov, shape=(m, n))
+
+    Above, the ith row in vals has a variance that is scaled by 4^i.
+    Alternatively, row or column cholesky matrices could be substituted for
+    either covariance matrix. The MatrixNormal is quicker way compute
+    MvNormal(mu, np.kron(rowcov, colcov)) that takes advantage of kronecker product
+    properties for inversion. For example, if draws from MvNormal had the same
+    covariance structure, but were scaled by different powers of an unknown
+    constant, both the covariance and scaling could be learned as follows
+    (see the docstring of `LKJCholeskyCov` for more information about this)::
+
+        # Setup data
+        true_colcov = np.array([[1.0, 0.5, 0.1],
+                              [0.5, 1.0, 0.2],
+                              [0.1, 0.2, 1.0]])
+        m = 3
+        n = true_colcov.shape[0]
+        true_scale = 3
+        true_rowcov = np.diag([true_scale**(2*i) for i in range(m)])
+        mu = np.zeros((m, n))
+        true_kron = np.kron(true_rowcov, true_colcov)
+        data = np.random.multivariate_normal(mu.flatten(), true_kron)
+        data = data.reshape(m, n)
+
+        # Setup right cholesky matrix
+        sd_dist = pm.HalfCauchy.dist(beta=2.5, shape=3)
+        colchol_packed = pm.LKJCholeskyCov('colcholpacked', n=3, eta=2,
+                                           sd_dist=sd_dist)
+        colchol = pm.expand_packed_triangular(3, colchol_packed)
+
+        # Setup left covariance matrix
+        scale = pm.Lognormal('scale', mu=np.log(true_scale), sd=0.5)
+        rowcov = tt.nlinalg.diag([scale**(2*i) for i in range(m)])
+
+        vals = pm.MatrixNormal('vals', mu=mu, colchol=colchol, rowcov=rowcov,
+                               observed=data, shape=(m, n))
+    """
+
+    def __init__(self, mu=0, rowcov=None, rowchol=None, rowtau=None,
+                 colcov=None, colchol=None, coltau=None, *args, shape=None,
```

Comment:
The python 2 travis tests point to this line as causing an error:

    File "/home/travis/build/pymc-devs/pymc3/pymc3/distributions/multivariate.py", line 1183
        colcov=None, colchol=None, coltau=None, *args, shape=None,
                                                       ^
    SyntaxError: invalid syntax

I think it should be `shape=None, *args, **kwargs`.